packages/move/tests/test_end_to_end.move

```diff
@@ -62,6 +62,13 @@ module deployment_addr::test_end_to_end {
         let user1_balance = primary_fungible_store::balance(user1_addr, fa_metadata_object);
         assert!(user1_balance == 0, user1_balance);
 
+        // FIX: for more thorough testing you'll want to add other assertions such as VestingStream, etc state being initialized correctly
+        // and that the relevant fungible store increased it's holding appropriately.
```

Comment:
Agreed, I should have included these assertions as well. I do test the initialized state of the VestingStream via the view functions test `test_view_functions` later on. But especially the check on the fungible store balances would be good to include in other tests.